js/altScheduleGenerator.js

```diff
@@ -35,7 +35,7 @@ function parseAlternate (summary, description) {
 
       if (!times) {
         if (periods.length > 0) {
-          periods[periods.length - 1].original += '\n' + name
+          periods[periods.length - 1].name += '\n' + name
```

Comment:
This fixes the bug where `,"original":"undefined\n"` takes up like 27% of the alternate schedule data. I guess I renamed it partway and never updated it